airflow/www/views.py

```diff
@@ -1818,7 +1823,12 @@ def index(self):
         session.close()
 
         # get a list of all non-subdag dags visible to everyone
-        unfiltered_webserver_dags = [dag for dag in dagbag.dags.values() if not dag.parent_dag]
+        # optionally filter out "paused" dags
+        if request.args.get('showPaused') == 'True':
+            unfiltered_webserver_dags = [dag for dag in dagbag.dags.values() if not dag.parent_dag]
```

Comment:
if we're adding yet another filter to the existing ones we need to rename the variables `do_filter`, `unfiltered_webserver_dags` etc